presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/TestPinotSplitManager.java

```diff
@@ -149,6 +149,15 @@ private void testBrokerTopNLarge(PinotTableHandle table, int sessionTopNLarge, i
         assertEquals(Integer.parseInt(limits[1]), configTopNLarge);
     }
 
+    @Test
+    public void testGetTimeBoundaryForTable()
+    {
+        assertEquals(pinotConnection.getTimeBoundary("hybrid").getOfflineTimePredicate().get(), "secondsSinceEpoch < '4562345'");
```

Comment:
Can you please add a test case with space in it, even though the current tests show single quotes.

Reply:
added! Thanks!